types/nock/index.d.ts

```diff
@@ -23,6 +23,7 @@ declare namespace nock {
     function isActive(): boolean;
     function isDone(): boolean;
     function pendingMocks(): string[];
+    function activeMocks(): string[];
```

Comment:
Docs: https://github.com/nock/nock#activemocks

This line needs to be duplicated on the `Scope` interface too. (after line 84 below)